### PR TITLE
loader added 

### LINKED ISSUE #193: add loader
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

### PR DESCRIPTION
**Pull request by adhavalboy**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-loader added
-some CSS modification
-some separated alert and function js file 

Fixes #193

### CODE REVIEW

File: static/js/functions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/*add all function script here*/
+
+/*<----start loader script---->*/
+$("nav").hide();
+				$(document).ready(function(){
+					$("nav").show();
+			 $(".welcome").fadeOut("slow", function() { $(this).css({visibility: "hidden", opacity: 0})});
```
[the remark this answers is not in the document]
Comment (adhavalboy) [reply static/js/functions.js]: where ?

[the remark this answers is not in the document]
Comment (adhavalboy) [reply static/js/functions.js]: done !

**Comment by meets2tarun** [comment]
Hi @adhavalboy 
Thanks for the PR.

Please file an issue for the PR and refer it in the fixes# in the PR description.

**Comment by adhavalboy** [comment]
done!

**Comment by meets2tarun** [comment]
@adhavalboy 
Thanks for doing so.

Please setup Heroku through https://github.com/fossasia/meilix-generator/blob/master/docs/installation/heroku.md and provide a link for reviewing the PR.

**Comment by meets2tarun** [comment]
@adhavalboy 
Would you like to resolve the existing conflicts?
You can ask for any help incase you need.

Thanks

**Comment by adhavalboy** [comment]
i want help for 
-delete static/js/email_noti.js
and idk why static/css/style.css is conflicting !!

**Comment by meets2tarun** [comment]
@adhavalboy 
Please try to pull master into your forked. Thus you will see conflicts in `static/css/style.css` and 
`static/js/email_noti.js` and then make the required changes and remove the unnecessary things.

Please tell does this make sense to you?

**Comment by adhavalboy** [comment]
Thanks for the help finally work out !! here is latest deployed https://still-inlet-48053.herokuapp.com

**Comment by meets2tarun** [comment]
@adhavalboy 
Would you please resolve the conflict present in `static/js/email_noti.js`?
then we can easily ship this.

Thanks

**Comment by tabesin** [comment]
Guys, please consider the general line try to make meilix generator into a flask app consistent with the other similar apps of Fossasia, and keep a low profile on monkey patching and js. Of course this could be merged anyways.

**Comment by tabesin** [comment]
Furthermore, please keep the scope per PR small. Not too many files and changes.
"Pull early pull often" ;)

**Pull request merged**